go.sum

```diff
@@ -1,8 +1,7 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
-decred.org/cspp v0.3.0 h1:2AkSsWzA7HIMZImfw0gT82Gdp8OXIM4NsBn7vna22uE=
```

Comment:
Something is wrong here. Even when I bump both dcrwallet and dcrctl, my `go mod tidy` isn't removing all of these other lines as well.  If you can't figure out why this is and fix it, it's fine to not run `tidy` at all.

Reply:
The extra lines were removed because I ran `rm go.sum; go mod tidy`. Adding them back now